README.md

```diff
@@ -1,15 +1,22 @@
 # React MonoRepo
 
-This repo contains react apps and component libraries to share between apps.
+This repo contains react demo apps and component libraries to share between apps.
 
 The repo uses [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) to link dependencies, and to enable running `yarn install` once.
 
 **You should use yarn instead of npm.**
 
 
 ## Projects
-* **mc-conflict-ui** - UI app for resolving conflicts.
-* **mc-react-components** - Component library for shared components.
+* **demoapp-react** - UI app as a demo for using react components.
```

Comment:
it's an old code, merge from master is needed

Reply:
done